frontend/institution/institutionHeader.component.js

```diff
@@ -32,8 +33,9 @@
             /** Return the title of page according current state.
              */
             instHeaderCtrl.getTitle = function getTitle(){
+                const instName = (instHeaderCtrl.actionsButtons) && instHeaderCtrl.actionsButtons.getLimitedName(110);
```

Comment:
use lodash get. so it will be like this `_.get(instHeaderCtrl.actionsButtons.getLimitedName(110))`

Reply:
_.get(instHeaderCtrl, 'actionsButtons.getLimitedName(110)')